content/en/security/guide/security-findings-migration.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Migrating to the New Security Findings Data Model
+description: Learn how to migrate to the unified Security Findings query syntax and data model, which consolidates querying across Cloud Security and Code Security findings.
+private: true
+further_reading:
+- link: "/security/cloud_security_management/"
+  tag: "Documentation"
+  text: "Cloud Security"
+- link: "/security/code_security/"
+  tag: "Documentation"
+  text: "Code Security"
+- link: "/security/application_security/"
+  tag: "Documentation"
+  text: "Application Security"
+---
+
+## Overview
+
+The way Security Findings are queried is changing and it may impact your workflows. You notice changes to how queries are constructed in Datadog. A set of [new features](#new-features) exposing that new schema is also released as a part of the upgrade.
+
+## Required action
+
+If you are using {insert relevant APIs / Terraform resource}, plan to migrate to the new version by {deprecation date}.
+
+Configuration in Datadog is updated automatically, so no action is needed there.
+
+## What are Security Findings
+
+Security Findings encompass vulnerabilities, misconfigurations, and security risks identified across your infrastructure and applications.
+
+### Code Security Findings
+
+- Library Vulnerabilities (SCA)
+- Static Code Vulnerabilities (SAST)
+- Secrets
+- Runtime Code Vulnerabilities (IAST)
+- Infrastructure as Code
+
+### Cloud Security Findings
+
+- Misconfigurations (CSPM)
+- Host & Container Vulnerabilities
+- Identity Risks (CIEM)
+- Attack Paths
+
+API security findings are also produced by App & API Protection (in preview).
+
+## What is changing
+
+### New querying syntax
+
+Following this change, all Security Findings start being queried the same way, using shared syntax and attributes organized around namespaces. Here are some practical examples:
+
+| Before | After |
+|--------|-------|
+| **Misconfigurations:** `@workflow.triage.status:open status:critical`<br>**Library vulnerabilities:** `status:open severity:Critical` | **All findings:** `@status:open @severity:critical` |
+| **Misconfigurations:** `@dd_computed_attributes.is_publicly_accessible:true`<br>**Host Vulnerabilities:** `is_publicly_accessible:Accessible` | **All findings:** `@risk.is_publicly_accessible:true` |
+| **Library Vulnerabilities:** `library_name:org.apache.logging.log4j`<br>**Host Vulnerabilities:** `package:org.apache.logging.log4j` | **All findings:** `@package.name:org.apache.logging.log4j` |
```

Comment:
```suggestion
| Before | After (all findings) |
|--------|----------------------|
| **Misconfigurations:** `@workflow.triage.status:open status:critical`<br>**Library vulnerabilities:** `status:open severity:Critical` | `@status:open @severity:critical` |
| **Misconfigurations:** `@dd_computed_attributes.is_publicly_accessible:true`<br>**Host Vulnerabilities:** `is_publicly_accessible:Accessible` | `@risk.is_publicly_accessible:true` |
| **Library Vulnerabilities:** `library_name:org.apache.logging.log4j`<br>**Host Vulnerabilities:** `package:org.apache.logging.log4j` | `@package.name:org.apache.logging.log4j` |
```
Thought we could put "all findings" into the column header so we didn't have to repeat it 🙂